glide-core/redis-rs/redis/src/commands/cluster_scan.rs

```diff
@@ -632,12 +740,16 @@ mod tests {
     }
     // Create a mock connection
     struct MockConnection;
-    #[async_trait]
     impl ClusterInScan for MockConnection {
         async fn refresh_if_topology_changed(&self) -> RedisResult<bool> {
             Ok(true)
         }
-        async fn get_address_by_slot(&self, _slot: u16) -> RedisResult<Arc<String>> {
+        async fn get_address_by_slot(
```

Comment:
I think that `trait` are allowed to have default implementation - why not move this behavior to the trait definition and override only the ones you need?

Reply:
removed the trait